corehq/apps/analytics/views.py

```diff
@@ -32,7 +33,11 @@ def dispatch(self, request, *args, **kwargs):
         return super(GreenhouseCandidateView, self).dispatch(request=request, args=args, kwargs=kwargs)
 
     def post(self, request, *args, **kwargs):
-        digester = hmac.new(settings.GREENHOUSE_API_KEY, request.body, hashlib.sha256)
+        digester = hmac.new(
+            settings.GREENHOUSE_API_KEY.encode('utf-8')
+            if isinstance(settings.GREENHOUSE_API_KEY, six.text_type) else settings.GREENHOUSE_API_KEY,
```

Comment:
Might be worth making an abstraction for this so it's not so imposing, like `utf8_bytestring()` or `coerce_to_bytestring()` - I've seen this same logic in a few places now.